### PR TITLE
Move wallet info out of navbar into main screen

### LINKED ISSUE #519: Direct link to wallet by clicking on balance
This will help "finding" the wallet.

### LINKED ISSUE #536: Balance is not centered
The majority of the UI uses a centered layout, but the balance is not centered.

image

### PR DESCRIPTION
**Pull request by bonomat**
resolves #519
resolves #536

image

### CODE REVIEW

File: taker-frontend/src/components/Wallet.tsx [hunk 1]
```diff
@@ -49,19 +50,27 @@ export default function Wallet(
...
         <HStack>
-            <Text align={"left"} as="b">On-chain balance:</Text>
+            <Text align={"left"} as="b">Wallet Balance:</Text>
             <Skeleton isLoaded={balance != null}>
-                <Text>{balance} BTC</Text>
+                <HStack>
+                    <Text>{balance} BTC</Text>
+                    <IconButton
+                        bg={"transparent"}
+                        aria-label="Goto wallet"
```
Comment (bonomat) [inline taker-frontend/src/components/Wallet.tsx:66]:
```suggestion
                        aria-label="Go to wallet"
```

Comment (1010Tom) [inline taker-frontend/src/components/Wallet.tsx]:
```suggestion
                        aria-label="Go to wallet"
```

**Review by 1010Tom — APPROVED** [review]
Looks good 👍

**Pull request merged**